packages/common/src/store/pages/audio-rewards/types.ts

```diff
@@ -28,6 +28,7 @@ export type UndisbursedUserChallenge = Pick<
   completed_blocknumber: number
   handle: string
   wallet: string
+  created_at: string
```

Comment:
should try to pass these through as camelCase, but i see that this is already a little wonk in this file

Reply:
I know it's a big lift, but would love to get auto-response transformation working at the SDK-ish level so that things go between camel/snake-case automatically.